web/rest/client/src/Model/ProfileAcl.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Model;
+
+use Assert\Assertion;
+use Ivoz\Api\Core\Annotation\AttributeDefinition;
+use Ivoz\Provider\Domain\Model\AdministratorRelPublicEntity\AdministratorRelPublicEntity;
+use Ivoz\Provider\Domain\Model\AdministratorRelPublicEntity\AdministratorRelPublicEntityInterface;
+use Ivoz\Provider\Domain\Model\Company\CompanyInterface;
+use Ivoz\Provider\Domain\Model\PublicEntity\PublicEntityInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class ProfileAcl
+{
+    /**
+     * @var string
+     * @AttributeDefinition(type="string")
+     */
+    private $iden;
+
+    /**
+     * @var bool
+     * @AttributeDefinition(type="bool")
+     */
+    private $create = false;
+
+    /**
+     * @var bool
+     * @AttributeDefinition(type="bool")
+     */
+    private $read = false;
+
+    /**
+     * @var bool
+     * @AttributeDefinition(type="bool")
+     */
+    private $update = false;
+
+    /**
+     * @var bool
+     * @AttributeDefinition(type="bool")
+     */
+    private $delete = false;
+
+    public function __construct(
+        AdministratorRelPublicEntityInterface $administratorRelPublicEntity
+    ) {
+        $publicEntity = $administratorRelPublicEntity->getPublicEntity();
+
+        $this->iden = $publicEntity->getIden();
+        $this->create = $administratorRelPublicEntity->getCreate();
+        $this->read = $administratorRelPublicEntity->getRead();
+        $this->update = $administratorRelPublicEntity->getUpdate();
+        $this->delete = $administratorRelPublicEntity->getDelete();
+    }
+
+    public function getIden(): string
+    {
+        return $this->iden;
+    }
+
+    public function isCreate(): bool
```

Comment:
Beign ACLs, I think `canCreate`, `canRead`, etc may be most understandable naming for these method